pydatarecognition/runcontrol.py

```diff
@@ -0,0 +1,298 @@
+"""Run Control object for regolith
```

Comment:
put text here saying it was copied from regolith etc. but is runcontrol for pydr etc etc.